lib/globals.js

```diff
@@ -108,6 +108,18 @@ let globals = {
         }
         items = items.map((item, index) => {
             if (typeof item === 'string') item = { value: item };
+
+            if (item.divider) {
+                if (typeof item.divider !== 'string') {
+                    let key = item.key || [
+                        contentKey + '.divider.label',
+                        'fields.default.divider.label'
+                    ];
+                    item.divider = translate(key);
+                }
+                return item;
```

Comment:
Just to be clear, are we returning the `item` here because if its a divider, there's nothing else we can do with this data?

Reply:
The govuk component just renders a div with the text, it doesn't even support classes or IDs

https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/components/radios/template.njk#L56-L58

Reply:
How has this been tested?
I’m struggling to get it working on the Driving Licence FE. Is there an example of it in use?

Reply:
@MMcC-GDS Are you happy with this solution to your requirements?

Reply:
@merlinc yep, I'm happy for it to be merged if there are no issues with anyone else

See https://github.com/alphagov/di-ipv-cri-dl-front/pull/78/files